src/test/java/teammates/test/cases/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -134,15 +137,16 @@ public void testExceptionalCases() throws Exception {
 
         resultsPage = loginToInstructorFeedbackResultsPage("CFResultsUiT.SanitizedTeam.instr",
                                                            "Session with sanitized data");
-
+        clickAjaxLoadedPanelAndWaitForExpansion("panelHeading-1", "ajax_auto");
         resultsPage.verifyHtmlMainContent("/instructorFeedbackResultsPageWithSanitizedData.html");
     }
 
     private void testModerateResponsesButton() {
 
         resultsPage = loginToInstructorFeedbackResultsPage("CFResultsUiT.instr", "Open Session");
         resultsPage.displayByQuestion();
-
+        clickAjaxLoadedPanelAndWaitForExpansion("panelHeading-2", "ajax_auto");
```

Comment:
my bad, looks like in this test case the convention is that set up is done before the test case